### DISCUSSION TITLE
https://greendolphindance.com/2024/08/27/%E5%85%AB%E6%9C%88%E7%A2%8E%E7%A2%8E%E5%BF%B5/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#29] [General]
# https://greendolphindance.com/2024/08/27/%E5%85%AB%E6%9C%88%E7%A2%8E%E7%A2%8E%E5%BF%B5/

Back to the rough ground!

https://greendolphindance.com/2024/08/27/%E5%85%AB%E6%9C%88%E7%A2%8E%E7%A2%8E%E5%BF%B5/

**Comment by wang7peng** [comment]
这篇文章读起来~~ **软绵绵的**，有种我之怀矣自诒伊阻的思妇之意。

**Comment by HelloGakki** [comment]
少女情怀总是诗呀
